Makefile

```diff
@@ -253,4 +260,7 @@ hack: hack-push-images hack-build-cli
 		--set worker.image.pullPolicy=Always \
 		--set logger.linux.image.repository=$(DOCKER_IMAGE_PREFIX)logger-linux \
 		--set logger.linux.image.tag=$(IMMUTABLE_DOCKER_TAG) \
-		--set logger.linux.image.pullPolicy=Always
+		--set logger.linux.image.pullPolicy=Always \
+		--set gitInitializer.image.repository=$(DOCKER_IMAGE_PREFIX)git-initializer \
+		--set gitInitializer.image.tag=$(IMMUTABLE_DOCKER_TAG) \
+		--set gitInitializer.image.pullPolicy=Always
```

Comment:
Nit: If we move the `--set gitInitializer.image...` lines above the `--set worker.image...` lines, it keeps two related things closer together.

Reply:
Moved.